plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/DefaultJdbcMetadata.java

```diff
@@ -716,7 +717,10 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
     public Optional<ConnectorOutputMetadata> finishCreateTable(ConnectorSession session, ConnectorOutputTableHandle tableHandle, Collection<Slice> fragments, Collection<ComputedStatistics> computedStatistics)
     {
         JdbcOutputTableHandle handle = (JdbcOutputTableHandle) tableHandle;
-        jdbcClient.commitCreateTable(session, handle);
+        Set<Long> successfulPageSinkIds = fragments.stream()
```

Comment:
nit: you can extract:
```
fragments.stream()
                .map(slice -> slice.getLong(0))
                .collect(toImmutableSet());
``` 
to helper method `getSuccessfulPageSinkIds(Collection<Slice> fragments)`